shotover-proxy/src/config/topology.rs

```diff
@@ -355,7 +353,7 @@ redis_chain:
     async fn test_validate_chain_invalid_subchain_redis_cache() {
         let expected = r#"Topology errors
 redis_chain:
-  SimpleRedisCache:
+  RedisCache:
```

Comment:
nit: let's avoid renames in this PR

Reply:
The transform is already called RedisCache, I'm fixing a bug where shotover's error output referred to RedisCache as SimpleRedisCache.